utils/hv-tools/files/etc/init.d/hv_vss_daemon

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh /etc/rc.common
+
+START=21
+
+USE_PROCD=1
+NAME=hv_vss_daemon
```

Comment:
dtto

Reply:
The hv_vss_daemon is a service that is part of the Hyper-V Guest Tools and is responsible for the Volume Shadow Copy Service (VSS) integration between a guest virtual machine (VM) and the Hyper-V host.

What does hv_vss_daemon do?
Volume Shadow Copy Service (VSS) Integration:

The hv_vss_daemon facilitates the creation of consistent snapshots (backups) of the guest VM’s file system while it is running.
It interacts with the VSS on the guest operating system to ensure that snapshots or backups are made without interrupting the services or processes running on the VM.
Usage in Virtualized Environments:

This service is crucial for performing reliable backups of VMs. When the Hyper-V host initiates a backup, the hv_vss_daemon ensures that the backup is consistent by coordinating with the guest OS to momentarily pause file system operations.
This process helps in preventing data corruption or inconsistency in backups, which is essential for disaster recovery scenarios.
Integration with Hyper-V:

The hv_vss_daemon works closely with other Hyper-V Integration Services to maintain a smooth interaction between the guest VM and the host, ensuring that the VM’s state is properly captured during backups.
This integration is key to maintaining data integrity and ensuring that the VM can be restored from a backup without issues.
Why is hv_vss_daemon needed in OpenWrt?
Including this daemon in OpenWrt allows OpenWrt virtual machines running on Hyper-V to support consistent and reliable backups through the Hyper-V host. This capability is particularly important for ensuring data integrity and for the smooth recovery of OpenWrt VMs in case of failures or system restores.

Reply:
removed NAME=hv_vss_daemon